src/main/java/io/phdata/streamliner/schemadefiner/model/TableDiff.java

```diff
@@ -20,8 +20,8 @@
 public class TableDiff {
   private String type;
   public String destinationName;
-  public Boolean existsInDestination;
-  public Boolean existsInSource;
+  public boolean existsInDestination;
+  public boolean existsInSource;
```

Comment:
changed it to boolean from Boolean because default of boolean is false and these are used in templates. In case of initial script generation it will prevent schema evolve scripts to generate queries otherwise it will give errors because tableDiff is not present in initial runs.